package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "node esbuild.config.mjs",
     "build": "node esbuild.config.mjs production",
-    "lint": "eslint ./src --fix && eslint ./tests --fix && tsc --noEmit --pretty && svelte-check",
+    "lint": "markdownlint-cli2-fix \"**/*.md\" \"#node_modules\" && eslint ./src --fix && eslint ./tests --fix && tsc --noEmit --pretty && svelte-check",
+    "lint:markdown": "markdownlint-cli2-fix \"**/*.md\" \"#node_modules\"",
```

Comment:
Is it correct, that markdown linting now happens for both scripts, `lint` and `lint:markdown`?

Reply:
Good point. That does seem odd.